### PR TITLE
Mentorship at Artsy

### PR DESCRIPTION
**Pull request by erikdstock**
A start on a document about where to find success when mentoring a new hire.

### CODE REVIEW

File: onboarding/mentors.md [hunk 1]
```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
 description: How to be an effective mentor of a new hire
 ---
 
-TBD
+# Mentorship at Artsy
+
+### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
+- [Alan's post](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/)
```
Comment (sweir27) [inline onboarding/mentors.md]: #minor The link should describe what the article is about. So something like "Blog post on professional development at Artsy"

File: onboarding/mentors.md [hunk 2]
```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+
+## What makes a good mentor (Keep?)
+- A first stop for stupid questions
+
+## Mentorship from day 0
+- Keep tabs on your mentee's schedule. Their first day will be full of meetings with People Ops but some good tasks for a mentor[:
+  - Introduce yourself
+  - Set a time for an [introductory coffee or tea](#your-first-meeting)
+  - Get in touch with the mentee's manager
+  - (Re-)familiarize yourself with people resources across the company that will be important to your mentee:
```
Comment (sweir27) [inline onboarding/mentors.md]: Might be  nice to explicitly link to any relevant resources (like atlas, the onboarding guide, etc.)

File: onboarding/mentors.md [hunk 3]
```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+- Talk about your own history at and before joining Artsy
+- Where are they coming from?
+  - If they will be remote, where are they from? What is it like there & what works for them in terms of remote work?
+  - Are they new in town? Where do they live, for how long? Just have a normal conversation.
+- Prior experience with our technology stacks (and others)
+- Other relevant past experience
+- Are they using the [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)? If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good things to do in between People Ops meetings, while working nearby each other or to pair on.
+
+### Your first week
+- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
```
Comment (sweir27) [inline onboarding/mentors.md]: Worth noting that the mentor should encourage the new hire to update  anything that is out-of-date 😄

File: onboarding/mentors.md [hunk 4]
```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+- Look for chances to pair or work alongside one another throughout the week.
+  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
+  - If they haven't set up hokusai yet, consider pairing on that.
+- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit things you talked about in the first meeting and look for areas that could have gone better. Consider amending relevant documents like this one.
+
+### Pairing (Keep?)
```
Comment (sweir27) [inline onboarding/mentors.md]: Yeah keep! Especially for the time before they are on a sprint rotation.

File: onboarding/mentors.md [hunk 5]
```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+
+### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
+- [Alan's post](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/)
+
+## Why
+One of Artsy's core values is that [People are Paramount](#). Artsy's engineering team works to reflect this in practice...
+
+## What makes a good mentor (Keep?)
+- A first stop for stupid questions
```
Comment (sweir27) [inline onboarding/mentors.md]: 🙏 yes keep! And maybe stick stupid in quotes since no questions  are truly stupid 😁

**Comment by erikdstock** [comment]
Updated with a more complete document. I still need to address some feedback above and fill in some links but I feel like the overall structure is now there.

File: onboarding/mentors.md [hunk 6]
```diff
@@ -3,4 +3,79 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+
+What does this look like in practice? In an ideal world it would be automatic and spontaneous between every member of the team. Some of our [engineering principles](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#readme) appeal explicitly to this ideal - **Psychological Safety**, **Leverage Your Impact**, **De-silo Engineers** and **Being Nice is Nice** stand out - while others like **Open Source by Default** and **Incremental Revolution**, while technical on the surface have clear analogues in how we work together. In practice we understand that all of these ideals require intentional work and iteration between people to make them reality. Artsy mentors, assigned when a new engineer joins the team, work to bridge that gap.
+
+## What makes a good mentor
+The role of a mentor is purposely different from that of a boss, a manager or a technical lead, and that role shifts to accommodate the needs of a team member. There may be multiple overlapping descriptions - a mentor may be someone who
```
Comment (sweir27) [inline onboarding/mentors.md]: #minor should we just say `from that of a manager or technical lead`?

File: onboarding/mentors.md [hunk 7]
```diff
@@ -3,4 +3,79 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+- *Facilitates onboarding*: Simply someone who can help a new hire smooth out the bumps of getting onboarded.
+- *Knows how things work*: From a non-judgemental sounding board for ['stupid' questions](#simple-answers-to-common-questions) to someone who can simply help find the best point person or stakeholder for a project.
+- *Helps develop people skills*: Artsy's engineering and company cultures are specific and open, but there is always room for improvement and new perspectives, especially the team changes. A mentor can act as a conduit between their mentee and the team as a whole.
```
Comment (sweir27) [inline onboarding/mentors.md]: Curious what you mean by this point! (Do you have an example?)

Comment (orta) [reply onboarding/mentors.md]: I wonder if the example can be framed as highlighting successes?

File: onboarding/mentors.md [hunk 8]
```diff
@@ -3,4 +3,79 @@ title: Onboarding Responsibilities for Mentors of New Hires
...
+- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of 2 months. It's good to check in with your mentee during this time.
+- Consider reaching out to other mentors as well
+- Periodically check in with yourself on what you think you know about your mentee:
+  - How are they feeling at work? It's good to be aware (if you can) of comfort level, new technologies and more dynamic pieces of how they are getting situated at Artsy.
+  - Are there any opportunities to pair together (or for them to pair with someone else) on a new technology or domain area?
+- Many mentoring pairs continue scheduling semi-weekly coffee chats in the weeks and months going forward. As the relationship becomes less formal these can be great to keep in touch with a team member you have a special relationship with!
+
+## Footnotes
+#### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
```
Comment (orta) [inline onboarding/mentors.md]: > - At Artsy we offer each new team members a spirit guide— a tenured team member who works in a different part of the company and support the onboarding employee’s transition to Artsy.  Who listens, offers advice and helps new teammates feel comfortable in their new work environment. 

From 🔒  [a blog post we never shipped on mentorship](https://docs.google.com/document/d/1LjO8snEL5RB0fKDUd6TqufllJdqyh_dFmibZaXduhV4/edit?usp=sharing) /cc @ashfurrow  who probably also forgot about this

Comment (ashfurrow) [reply onboarding/mentors.md]: I did forget about this, yeah!

**Comment by erikdstock** [comment]
merge on green

File: onboarding/mentors.md (hunk 7 above)
Comment (erikdstock) [reply onboarding/mentors.md]: I removed this point since it felt very redundant, but followups to this document are great.

**Pull request merged**